templates/CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# How to contribute
+
+Thanks for showing interest and sharing your time to contribute to this project!
+
+This guide is meant to be used as a general guideline for creating issues or
+pull requests. We encourage all first contributors to give this a read to avoid
+common mistakes and improve the quality of all contributions.
+
+## Opening issues
+
+Before creating a new issue make sure you use the search functionality to confirm
+that a similar issue doesn't already exist. Next, make sure you properly label
+the issue as per our [labels](https://github.com/trento-project/projectname/labels)
+
+If you are reporting a `bug`, please share a file generated using the
+`trento-support.sh` script with the following params:
+
+```
+# trento-support.sh --collect all --output file-tgz
+```
+
+and include the output to your issue. The script should remove sensitive data
+automatically but please make sure you are not sharing anything sensible yourself.
```

Comment:
..but please make sure you are not sharing **any sensitive data of your own**.
(Suggestion highlighted in bold)

Reply:
@jagabomb thanks, addressed